apps/src/music/player/ToneJSPlayer.ts

```diff
@@ -225,29 +225,29 @@ class ToneJSPlayer implements AudioPlayer {
       );
     });
 
-    // Create a clock that ticks forward every 16th note, and stops when the sequence finishes.
```

Comment:
`Tone.Clock` had some idiosyncrasies that made the calling code complicated, especially in tight timing scenarios like when a preview was stopped but another one begun almost immediately.  Most notably, there would be a delay after attempting to stop the clock before it would actually stop and fire the `on('stop')` handler.  And sometimes, it would generate a timed callback even after it had called the `on('stop`)` handler.  For the former, the calling code would need additional statefulness to hold off starting a new preview until the old one had definitely finished.

Switching to a new implementation with `setInterval` resolved a variety of edge-case errors, and allowed the calling code to authoritatively control the state of the timer, simplifying the calling code as well.